client/app/components/dynamic-form/DynamicFormField.jsx

```diff
@@ -50,10 +48,23 @@ export const FieldType = PropTypes.shape({
   props: PropTypes.object, // eslint-disable-line react/forbid-prop-types
 });
 
+const FieldTypeComponent = {
+  checkbox: CheckboxField,
+  file: FileField,
+  select: SelectField,
+  number: NumberField,
+  textarea: TextAreaField,
+  ace: AceEditorField,
+};
+
 export default function DynamicFormField({ form, field, ...otherProps }) {
-  const { name, type, readOnly, autoFocus, initialValue } = field;
+  const { name, type, readOnly, autoFocus } = field;
   const fieldLabel = getFieldLabel(field);
 
+  if (type === "content") {
+    return field.content;
+  }
```

Comment:
Can it be a "field" component as well or there are some limitations in this case?

Reply:
It was mostly to have it directly, without a component wrapping it, but I realized that doesn't make sense, so added the ContentField :)